source/index.md

```diff
@@ -0,0 +1,69 @@
+<style>
```

Comment:
I left it here on purpose for 2 reasons:

1. I don't want to pollute the styles from Estatico and/or the project, specially considering that adding those styles in there will affect for sure modules and/or pages using in this case quotes.
1. I think it is great to have it as an example here to see that Estático is also capable to show beautiful documentation

What do you think?